lib/web/apiserver.go

```diff
@@ -2691,16 +2691,23 @@ func (h *Handler) siteNodeConnect(
 	h.log.Debugf("New terminal request for server=%s, login=%s, sid=%s, websid=%s.",
 		req.Server, req.Login, sessionData.ID, sessionCtx.GetSessionID())
 
-	authAccessPoint, err := site.CachingAccessPoint()
-	if err != nil {
-		h.log.WithError(err).Debug("Unable to get auth access point.")
-		return nil, trace.Wrap(err)
-	}
+	keepAliveInterval := req.KeepAliveInterval
```

Comment:
I'm unfamiliar with this code - is it possible for this keep alive interval to be really small but non-zero, e.g. 1 nanosecond? If so: does this pose any risk of a denial of service or other disruptive effect?

Alternatively, could we just check that the keep alive interval is above some minimum other than 0?

Reply:
I set the min interval to 1 second.